README.md

```diff
@@ -350,10 +354,46 @@ The generated parameter value for the nested map example can then be accessed wi
 
 ### Use generated struct in Cpp
 The generated header file is named based on the target library name you passed as the first argument to the cmake function.
-If you specified it to be `turtlesim_parameters` you can then include the generated code with `#include "turtlesim_parameters.hpp"`.
+If you specified it to be `turtlesim_parameters` you can then include the generated code with `#include <turtlesim/turtlesim_parameters.hpp>`.
 ```c++
-#include "turtlesim_parameters.hpp"
+#include <turtlesim/turtlesim_parameters.hpp>
 ```
+
+Note that this header can also be used from another package:
+```cmake
+cmake_minimum_required(VERSION 3.8)
+project(my_other_package)
+
+
+include(GNUInstallDirs)
```

Comment:
is this standard in ROS boilerplate cmakelists.txtx?

Reply:
Not really, this is a standard cmake module allowing to use GNU standard installation directories instead of manually defined ones, through cmake variables. This is not necessary for a minimal example, but I would deem it a good practice.

Reply:
Sounds good to me, thank you.